src/Frontend/Core/Engine/Block/ExtraInterface.php

```diff
@@ -149,6 +143,10 @@ public function execute(): void
     public function getAction(): ?string
     {
         if ($this->action !== null) {
+            if (!\in_array($this->action, $this->config->getPossibleActions(), true)) {
```

Comment:
what's the backslash for?

Reply:
opcode optimalisations https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/3048

Reply:
moh, nice, didn't know that